src/OrchardCore.Modules/OrchardCore.Contents/Views/Items/ContentsAdminFilters-Stereotype.Thumbnail.cshtml

```diff
@@ -0,0 +1,13 @@
+@model ShapeViewModel<ContentOptionsViewModel>
+@{
+    var term = Model.Value.FilterResult.FirstOrDefault(x => x.TermName == "stereotype");
+}
+
+<h4 class="card-title">Content Type Stereotype</h4>
+<pre class="mb-3">@(@term != null ? @term.ToString() : "stereotype:...")</pre>
```

Comment:
```suggestion
<pre class="mb-3">@(term?.ToString() ?? "stereotype:...")</pre>
```

Reply:
Yes, I just copy pasted from the other existing thumbnail shapes, but I will apply the suggestion ;)